From ea7b7e7e3a5c2881df30ad6c7962b087f09a5c7f Mon Sep 17 00:00:00 2001
From: JD Lien <jd@jdlien.com>
Date: Mon, 19 Oct 2026 15:59:06 -0600
Subject: [PATCH 1/4] PREVIEW, not for ak820pro-jdlien: Lights out's bulb with
 a 2 px slash through it (the 'turned off' idiom of a muted bell), slate like
 its outline, from above the glass's top left to beside the collar; to judge
 on the panel against the plain unlit bulb (7df85fb89b)

---
 keyboards/a_jazz/ak820pro/graphics/display.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/keyboards/a_jazz/ak820pro/graphics/display.c b/keyboards/a_jazz/ak820pro/graphics/display.c
index cc9a05a81c..762b2e7cdc 100644
--- a/keyboards/a_jazz/ak820pro/graphics/display.c
+++ b/keyboards/a_jazz/ak820pro/graphics/display.c
@@ -1639,10 +1639,10 @@ static const char bulb_art[POWER_MODE_COUNT][BULB_H][BULB_W + 1] = {
         "..#*******#..", "...#*****#...", "....#***#....", "....=====....",
         "....+++++....", ".....+++.....",
     },
-    [POWER_MODE_MINIMAL] = {
-        ".............", ".............", ".............", ".....###.....",
-        "...##...##...", "...#.....#...", "..#.......#..", "..#.......#..",
-        "..#.......#..", "...#.....#...", "....#...#....", "....=====....",
+    [POWER_MODE_MINIMAL] = {   /* PREVIEW: unlit, with a slash through it */
+        ".............", ".............", ".##..........", "..##.###.....",
+        "...##...##...", "...###...#...", "..#..##...#..", "..#...##..#..",
+        "..#....##.#..", "...#....##...", "....#...###..", "....=====.##.",
         "....+++++....", ".....+++.....",
     },
 };
-- 
2.54.0

